src/openqaoa-braket/openqaoa_braket/backends/qaoa_braket_qpu.py

```diff
@@ -85,9 +86,12 @@ def __init__(
                 else list(range(self.n_qubits))
             )
         else:
-            warnings.warn(
-                "Ignoring the initial_qubit_mapping since the routing algorithm chose one"
-            )
+            if isinstance(initial_qubit_mapping, list):
+                warnings.warn(
+                    "Ignoring the initial_qubit_mapping since the routing algorithm chose one"
```

Comment:
Is the message correct? What if I pass an initial mapping?

Reply:
But the warning says that the routing chooses it: what if I have passed the initial_qubit_mapping as a list at the very beginning? 

I guess I just wanted to check whether the text is consistent with this edge case!

Reply:
The SWAP sequence depends on the qubit mapping. So once the circuit is routed if you choose a different initial_mapping, the circuit will be invalid. Since we decided that the routing algorithm will also choose the initial_mappging, it will be ignored if the user specifies something different.